infrastructure/proto/log_replication_transport.proto

```diff
@@ -23,4 +23,6 @@ service LogReplication {
     // Used only by Sink when connection starter. This RPC creates a long living stream, on which the source and
     // sink exchange negotiation-msg/replication-msg/ACKs
     rpc reverseReplicate(stream org.corfudb.runtime.ResponseMsg) returns (stream org.corfudb.runtime.RequestMsg) {}
+
+    rpc sinkSideSessionInitialize(stream org.corfudb.runtime.RequestMsg) returns (org.corfudb.runtime.ResponseMsg) {}
```

Comment:
this RPC need not be cient streaming.
The defination shold be like the leadership or the negotiate msg.

Also, can we rename this to be source/sink agnotistice? Something like remoteSessionRegister or something similar